scottbarnesg/smart-sec-cam
# Fix CORS issue with self-signed certificates

##### Changes:

- Updates Flask server to serve React app.
- Removes unused nginx config
- Moves `frontend` container to an intermediate container in `server`
- Updates default server port for both the UI and API to `8443`

##### Testing:

- Check out this branch
- Generate new (untrusted) certs: `./create-certs.sh`
- Start the server: `docker-compose up -d --build`
- Open the Web UI in Firefox at `https://localhost:8443`. After clicking through the certificate warnings, open the debug console and verify there are no CORS errors.
- Navigate to the videos page: `https://localhost:8443/videos`. Verify the videos load correctly and that there are no CORS errors in the debug console. 

Closes #16
